toolkit/docs/building/building.md

```diff
@@ -214,13 +214,13 @@ All images are generated in the `out/images` folder.
 ### Virtual Hard Disks and Containers
 
 ```bash
-# To build a Mariner VHD Image (VHD folder: ../out/images/core-legacy)
+# To build a Azure Linux VHD Image (VHD folder: ../out/images/core-legacy)
 sudo make image CONFIG_FILE=./imageconfigs/core-legacy.json REBUILD_TOOLS=y
 
-# To build a Mariner VHDX Image (VHDX folder ../out/images/core-efi)
+# To build a Azure Linux VHDX Image (VHDX folder ../out/images/core-efi)
 sudo make image CONFIG_FILE=./imageconfigs/core-efi.json REBUILD_TOOLS=y
 
-# To build a core Mariner Contianer (Container Folder: ../out/images/core-container/*.tar.gz
+# To build a core Azure Linux Contianer (Container Folder: ../out/images/core-container/*.tar.gz
```

Comment:
replace "a" with "an" for the 3 changes.
Correct spelling of "container"

Reply:
Updated all occurences